src/Helper/AttributeHelper.php

```diff
@@ -61,17 +61,18 @@ public function __construct(
         $this->attributeValueRepositoryContract = $attributeValueRepositoryContract;
         $this->attributeValueNameRepositoryContract = $attributeValueNameRepositoryContract;
     }
-
+    
     /**
      * Get variation attributes.
      *
      * @param array $variation
 	 * @param KeyValue $settings
+	 * @param ElasticExportPropertyHelper $elasticExportPropertyHelper
      * @return array
      */
-    public function getVariationAttributes($variation, $settings):array
+    public function getVariationAttributes($variation, $settings, $elasticExportPropertyHelper):array
```

Comment:
Spricht was dagegen, dass in der Klasse gleich mit zu laden, anstatt als Param in der Funktion zu übergeben?

Reply:
Guter Einwand. Hätte Probleme mit dem cache gegeben. Habe das jetzt über singleton gelöst